<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="SingleWindow" name="tab_Single">
       <attribute name="title">
        <string>SINGLE</string>
       </attribute>
      </widget>
      <widget class="BatchWindow" name="tab_Batch">
       <attribute name="title">
        <string>BATCH</string>
       </attribute>
      </widget>
      <widget class="StitchWindow" name="tab_Stitch">
       <attribute name="title">
        <string>STITCH</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StitchWindow</class>
   <extends>QWidget</extends>
   <header>app</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SingleWindow</class>
   <extends>QWidget</extends>
   <header>app</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatchWindow</class>
   <extends>QWidget</extends>
   <header>app</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
